core/src/main/java/io/questdb/std/FilesFacade.java

```diff
@@ -64,6 +64,10 @@
 
     long getPageSize();
 
+    default long alignedOffset(long offset) {
```

Comment:
We have `Files.ceilPageSize()` method, this looks like should be `Files.floorPageSize()`

Reply:
Moved and renamed in 9678f2dec1966a917f63dd78de06e01c1d2d0740